app/strings.js

```diff
@@ -2,28 +2,46 @@ exports = typeof window === 'undefined' ? global : window;
 
 exports.stringsAnswers = {
   reduceString: function(str, amount) {
-    var count = 0;
-    var result = '';
+    var countedOccurences = 0;
+    var stringResult = '';
     for(var i = 0; i < str.length; i++) {
       if (str[i] === str[i+1]) {
-        count++;
-        if (count < amount) {
-          result = result + str[i];
+        countedOccurences++;
+        if (countedOccurences < amount) {
+          stringResult += str[i];
         }
       }
         else {
-          count = 0;
-          result = result + str[i];
+          countedOccurences = 0;
+          stringResult += str[i];
         }
       };
-      return result;
+      return stringResult;
   },
 
   wordWrap: function(str, cols) {
+    var formattedString = '',
+    arrayOfWords = [];
 
+    arrayOfWords = str.split(' ');
+
+    formattedString = arrayOfWords[0];
```

Comment:
here the same - you create on line 23 and then reassign on line 28, no reason to do it this way